js/src/popover.js

```diff
@@ -146,13 +145,7 @@ class Popover extends Tooltip {
 
   static jQueryInterface(config) {
     return this.each(function () {
-      let data = Data.get(this, DATA_KEY)
-      const _config = typeof config === 'object' ? config : null
-
-      if (!data) {
-        data = new Popover(this, _config)
-        Data.set(this, DATA_KEY, data)
```

Comment:
Why this `Data.set(this, DATA_KEY, data)` was required before? :confused:

Reply:
I really feel was a left over. The component initialize its self using the base-component constructor.
There the element is saved on js/src/base-component.js:34
```js
 Data.set(this._element, this.constructor.DATA_KEY, this)
```
line 154 seems like a duplication which re-saves the same thing